spring-integration-jdbc/src/main/java/org/springframework/integration/jdbc/store/channel/MessageRowMapper.java

```diff
@@ -51,6 +51,7 @@ public MessageRowMapper(AllowListDeserializingConverter deserializer) {
 	}
 
 	@Override
+	@SuppressWarnings("NullAway") // See https://github.com/uber/NullAway/issues/1075
```

Comment:
I'm not sure why do we need this?
Why our `mapRow()` just cannot be `@Nullable` like that `SingleColumnRowMapper` for `queryForObject(String, Class)`?